ui/app/appointments/constants.js

```diff
@@ -24,13 +25,17 @@ Bahmni.Appointments.Constants = (function () {
         defaultCalendarEndTime: "19:00",
         defaultExpandServiceFilter: -1,
         collapseServiceFilter: 0,
+        defaultWeekDays: [0, 1, 2, 3, 4, 5, 6],
```

Comment:
What does 0,1,2 .. mean - for defaultWeekDays and defaultWeekStartDay? if its supposed to be Sunday, Monday ...? I also see that below you have defaultWeekStartDayName as the day name. 
Why separate "defaultWeekStartDay" and "defaultWeekStartDayName"? 
Wouldn't it better, to create objects like :

const weekdays = {
    SUNDAY: 'SUNDAY',
    MONDAY: 'MONDAY',
    ....
}

like below. 

Seems to be redundant definition/usage of the same thing.

Reply:
This is angular calendar days of week. Changed the variable name to angularCalendarDaysOfWeek. We have to mention all the working days here. So it would be 0-6 all numbers.